components/formatter/PodImages.vue

```diff
@@ -25,6 +25,19 @@ export default {
       } else {
         return this.value;
       }
+    },
+    imageLabels() {
+      if (Array.isArray(this.images) && this.images.length > 1) {
+        let imagesNames = '<span style="text-decoration: underline">Images:</span><br/><br/>';
```

Comment:
This should be localised - alternatively, I think you could drop the title and just have the bulletted list, since the context of what you're seeing is obvious.